app/views/users/_create_form.html.erb

```diff
@@ -106,21 +116,15 @@
           <%= t('users._form.log_in') %>
         </a>
 
-      <script>
-        (function() {
-          $('.btn-save').click(function onClick(e) {
-            $(this).addClass("disabled") // disable the button after it is clicked
-                   .html("<i class='fa fa-spinner fa-spin'></i>"); // make a spinner that spins when clicked
-          })
-        })();
-      </script>
```

Comment:
Instead of removing this,  we can disable the button only when all the required fields are filled i.e., spinner will show only when all required fields are filled otherwise focus on the fields?

Reply:
Yes that would be ideal. Do you have any idea how this can be done ?. Because the problem is since button is a JS element and form validation happens in ruby, i wasn't able to find a way to check if all fields are filled before displaying the button. The fields i try to access show "Cant be accessed error on console". I'll keep trying other ways to do this in the meantime maybe i was wrong, Thanks @gauravano  :)

Reply:
The form/input fields we create in rails also result in the generation of HTML form only and so can be assessed just like that, open the sign-up form of publiclab.org and try `$('#username-signup').val()` in your console and you'll get the value of username field.

Reply:
Thanks, I'll try this :)